test/test_helper.rb

```diff
@@ -4,6 +4,8 @@
 
 module ActiveSupport
   class TestCase
+    include Devise::Test::IntegrationHelpers
```

Comment:
This imports the devise `sign_in` method that is used in the tests.

Reply:
Okay